diagnostic.md

```diff
@@ -40,31 +50,36 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+  The purpose of a serializer is to format the JSON response that the server will send to the client based on the data retrieved by the model.
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+    attributes :given_name, :surname, :email, :movies
+
+    def movies
+      object.movies.pluck(:id)
+    end
   end
   ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
   ```sh
-    # < Your Response Here >
+    bin/rails g scaffold favorite movie:references profile:references [rank:integer]
```

Comment:
The last argument will generate a migration that will throw errors when it tries to run.